### DISCUSSION TITLE
Need help with displaying curves from covid source

### DISCUSSION THREAD

**Discussion by UlrichThiess** [grafana/grafana-csv-datasource#38] [Q&A] [answered]
I use https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_confirmed_global.csv as source.

If i look into the table, i could see that the values are on x axis. How could i display the values with the graph panel? Or as sum in the worldmap panel?

Sorry but i need some help to start with it. Thanks a lot.

**Comment by marcusolsson** [comment, marked as answer]
This is a tricky problem that I don't yet know a solution for. Hopefully though this will make it into Grafana 8.0. You can find more information in [this issue](https://github.com/grafana/grafana/issues/22219#issuecomment-598160381).

If you do find a solution to this, I'd love to know :)
